src/form.spec.ts

```diff
@@ -253,7 +253,7 @@ describe(Form, () => {
       const form = createForm({
         validations: {
           emails: ({ model }) =>
-            model.emails.length === 0 ? ['specify at least one'] : [],
+            model.emails?.length === undefined ? ['specify at least one'] : [],
```

Comment:
this changes the meaning of this validation.
```suggestion
            (model.emails ?? []).length === 0 ? ['specify at least one'] : [],
```